bundles/nl.esi.comma.project.standard.cli/dist/simulator/CPNServer.py

```diff
@@ -165,3 +172,5 @@ def handle_markings_reload(uuid: str):
     # Setting host = "0.0.0.0" runs it on localhost
     app.run(host="0.0.0.0", debug=False)
 
+
+# TEMP_DIR.cleanup()
```

Comment:
The temp directory is not removed upon exit of the process. On my computer the following directory remains: `C:\Users\xxxxx\AppData\Local\Temp\cpnserver_20250410_133440358507_vk9f1mev`